packages/shared/src/utils/constants/idv-options.ts

```diff
@@ -1,3 +1,6 @@
 import { localize } from '@deriv/translations';
 
-export const IDV_NOT_APPLICABLE_OPTION = { id: '#NA', text: localize('I don’t have any of these') };
+/**
+ * Configuration that allows user to opt-out of IDV
+ */
+export const IDV_NOT_APPLICABLE_OPTION = { id: 'none', text: localize('I want to do this later'), value: 'none' };
```

Comment:
As we've a separate card just for this option renaming, we shouldn't rename in this card's scope

Reply:
I will remove this change from the other card as both these need to be sent on priority